src/network/websockets/server/index.ts

```diff
@@ -53,6 +53,25 @@ export function runWebsocketServer<AuthDetails>(params: Params<AuthDetails>) {
       socket.send(JSON.stringify(obj));
     };
 
+    let cachedMessages: Array<string> = [];
```

Comment:
`Array<string>` is perfectly fine, but we already use the shorter syntax in the repo anyway:

```suggestion
    let cachedMessages: string[] = [];
```